new-dti-website/src/app/utils.ts

```diff
@@ -0,0 +1,35 @@
+const populateMembers = (
+  roles: {
+    [key in Role]: {
+      roleName: string;
+      description: string;
+      members: IdolMember[];
+      order: string[];
+    };
+  },
+  allMembers: IdolMember[]
+) => {
+  for (const [key, value] of Object.entries(roles)) {
```

Comment:
Instead of the loop, try to do this more functionally with return ` Object.entries(roles).map(...)` instead of editing this in place, which makes it harder to follow (i.e. if this breaks, we'll have a harder time fixing it). Slack me if you need help!

This might involve changing more than just this function, but the refactor will be worth it for code readability.

![image](https://github.com/cornell-dti/idol/assets/59291082/13a71f62-d744-44bb-85c3-e3f660f8db95)